src/bridge/cockpitpeer.c

```diff
@@ -51,6 +53,12 @@ struct _CockpitPeer {
   /* Authorizations going on */
   GHashTable *authorizes;
 
+  /* authorize types we will reply to */
+  GHashTable *authorize_values;
```

Comment:
I'm going to try and change this authorize_values stuff into CockpitRouter + environment variable.